.github/tests.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Abstract Machines
+# SPDX-License-Identifier: Apache-2.0
+
+name: CI Pipeline
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  lint-and-build:
+    name: Lint and Build
+    runs-on: ubuntu-latest
```

Comment:
Modify it to use a different base OS something like https://github.com/0x6flab/dtop/blob/main/.github/workflows/ci.yaml.